lib/robots/dor_repo/gis_assembly/finish_gis_assembly_workflow.rb

```diff
@@ -14,15 +14,6 @@ def perform_work
           logger.debug "finish-gis-assembly-workflow working on #{bare_druid}"
           rootdir = GisRobotSuite.locate_druid_path bare_druid, type: :stage
 
-          # first ensure all files are ready
-          %w[
-            content/data.zip
-            content/preview.jpg
-          ].each do |f|
-            fn = File.join(rootdir, f)
-            raise "finish-gis-assembly-workflow: #{bare_druid} is missing required file: #{fn}" unless File.size?(fn)
-          end
-
```

Comment:
I think we removed checks like this from elsewhere in the robots. The only thing that would've been left was checking if `preview.jpg` was there, which I think we are doing in the `copy-data` step.